src/components/Table/Table.jsx

```diff
@@ -37,6 +37,9 @@ const getTable = (data, columns, sortState, sortField, sortClick) => (
           <MaterialTableRow
             key={index}
             className={styles.row}
+            style={{
```

Comment:
I think it's fine to keep empty rows.  Alternatively run the input data through a filter function?